config.js

```diff
@@ -5,7 +5,22 @@ const CONFIG = {
   [CONST.CONFIG.LOG_LEVEL_TELETEXT_SERVER]: CONST.LOG_LEVEL_VERBOSE,
   [CONST.CONFIG.LOG_LEVEL_TELETEXT_VIEWER]: CONST.LOG_LEVEL_VERBOSE,
 
-  [CONST.CONFIG.BASE_DIR]: '/var/www/',
+  // this is used by `update-service-pages.js` as the place where the
+  // raw source service pages directories (likely SVN repositories)
+  // are located.
+  // (Note: service pages are not actually served by Muttlee from here
+  //  they need additional renaming first, and are then served from
+  //  CONST.CONFIG.SERVICE_PAGES_SERVE_DIR defined below).
+  [CONST.CONFIG.SERVICE_PAGES_DIR]: '/var/www/teletext-services',
```

Comment:
@peterkvt80 feel free to commit a change to this if you want, this just seemed like a good location to me (one that can be mentioned in public documention guides in future)